src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,186 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connect a libSQL database for production
+
+Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server
+
+The way you get these values varies between services, but the next paragraph will show you how can you get started with [Turso](https://turso.tech), a fully managed database platform that uses libSQL.
+
+### Getting started with Turso
+
+You can either use the CLI or the web interface to create and rethrieve the values we need.
+The steps that are shown below will walk you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, getting the two values we need and pushing our schema to the remote database.
+
+<Steps>
+
+1. Install the [Turso CLI](https://docs.turso.tech/cli/installation).
+
+2. [Login or signup](https://docs.turso.tech/cli/authentication) to Turso. For this example the username for the account will be Houston.
```

Comment:
The username is not needed anywhere so I don't think that last bit is necessary.

It will appear as part of the URL, but that is an opaque value. Turso might as well give a random string or a hash, it doesn't change our guide here.
```suggestion
2. [Login or signup](https://docs.turso.tech/cli/authentication) to Turso.
```